pkg/oc/admin/prune/images.go

```diff
@@ -74,6 +76,10 @@ var (
 	  # To actually perform the prune operation, the confirm flag must be appended
 	  %[1]s %[2]s --keep-tag-revisions=3 --keep-younger-than=60m --confirm
 
+	  # To exclude an imagestream from pruning you can specify a regular expression
+	  # which will be applied to the '<namespace>/<name>:<tag>' string.
+	  %[1]s %[2]s --exclude-imagestreamtag=myproject/hello-openshift
```

Comment:
It may be worth mentioning the anchoring (^ and $).

Reply:
Or at least change the example to `--exclude-imagestreamtag=^myproject/hello-openshift:`

Reply:
And I'd change the option name `--keep-imagestreamtag=^myproject/hello-openshift:`